### PR TITLE
Add new schemes

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: app/constants/scheme-names.js [hunk 1]
```diff
@@ -17,5 +20,8 @@ module.exports = {
   [CS]: 'Countryside Stewardship',
   [BPS]: 'BPS',
   [FDMR]: 'FDMR',
-  [MANUAL]: 'Manual Invoice'
+  [MANUAL]: 'Manual Invoice',
+  [ES]: 'Environmental Stewardship',
+  [FC]: 'Forestry Commission',
+  [IMPS]: 'IMPS'
```
Comment (Kirsten1822) [inline app/constants/scheme-names.js:26]: I noticed in one of the other repos these were just CS, Manual, ES etc. Which is the preferred option?

Comment (johnwatson484) [reply app/constants/scheme-names.js:26]: Depends on what the values are used for.  The other one is the data hub which populates data in the dashboards where it was agreed to use the acronyms.

Here we're sending emails to the business and need to be a bit clearer about what the scheme is so we have the full name.

Comment (johnwatson484) [reply app/constants/scheme-names.js:26]: That is a fair comment.

BPS = Basic Payment Scheme
FDMR = Financial Discipline Mechanism Reimbursement

However, the business tend to refer to those commonly as BPS and FDMR, whereas they refer to CS as Countryside Stewardship.

So agree it's not consistent, but it makes sense from a business perspective.

Comment (Kirsten1822) [reply app/constants/scheme-names.js:26]: I'll only cry a little - that's alright if it's better for the business :)

**Review by Kirsten1822 — APPROVED** [review]
Happy to approve :)

**Pull request merged**